NEWS.md

```diff
@@ -2,11 +2,26 @@
 
 ## New features
 
-* `aes()` now supports quasiquotation so that you can use `!!`, `!!!`, and
-  `:=`. (ggplot2 does not currently support full tidy evaluation because
-  when I wrote ggplot2 my understanding of NSE was quite flawed, and hence
-  ggplot2 only captures one environment per plot, not one environment
-  per aesthetic. We will fix this in a future release.)
+* `aes()` now supports quasiquotation so that you can use `!!`, `!!!`,
+  and `:=`.
+
+* `facet_wrap()` and `facet_grid()` now support `vars()` inputs. Like
```

Comment:
I think this needs something to emphasise that `~` is not going away (i.e. it's not even soft-deprecated) but the new interface makes ggplot2 easier to program with.